features/integrate_build_system/waf-installer/README.md

```diff
@@ -0,0 +1,3 @@
+## How to Use
```

Comment:
Better to have a unified readme in the root of this example with all the explanations and general steps. The specific steps should be very straightforward to read from the launch scripts (.sh/.bat) IMO